ansible_rulebook/conf.py

```
@@ -38,6 +72,60 @@ def __init__(self):
+
+    @staticmethod
+    def _convert_type(value: Any, target_type: Type) -> Any:
+        """Convert value to target type"""
+        if isinstance(value, target_type):
+            return value
+
+        if target_type == bool:
+            if isinstance(value, str):
+                return value.lower() in ("true", "1", "yes", "on")
+            return bool(value)
+        elif target_type == list:
```

Ruff (0.15.15)

[error] Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)